tests/test_scripts.py

```diff
@@ -1016,6 +1016,66 @@ def test_load_graph():
     assert x == (1, 0), x
 
 
+def test_build_graph():
```

Comment:
most of these tests are copies of the tests that are being used on load-graph, yes? in which case they can be removed.